README.md

```diff
@@ -9,5 +9,13 @@ go build
 ./id-contact-auth-digid
 ```
 
+## Running tests
+This package contains tests, which can be run in the standard manner for go programs:
+```bash
+go test
+```
+
+Some tests depend on a postgres database, to run these, add the flags `-ldflags "-X github.com/id-contact/auth-digid.testdb=<DB URL>"` to the go test invocation, where `<DB URL>` should be the connection url for a postgres database that the tests have full rights to modify (including schema changes).
```

Comment:
I'd like to suggest adding this, to make it easy to run the tests:

For a quick setup using, the following command can be used:
```bash
docker run -e POSTGRES_PASSWORD=password -p 127.0.0.1:5432:5432 --rm postgres
```
Then, to run the tests:
```bash
go test -ldflags "-X github.com/id-contact/auth-digid.testdb=postgres://postgres:password@127.0.0.1:5432/postgres"
```